src/SourceLocator/SourceStubber/AggregateSourceStubber.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Roave\BetterReflection\SourceLocator\SourceStubber;
+
+use ReflectionClass as CoreReflectionClass;
+use ReflectionFunction as CoreReflectionFunction;
+
+class AggregateSourceStubber implements SourceStubber
+{
+    /**
+     * @var SourceStubber[]
+     */
+    private $sourceStubbers;
+
+    /**
+     * @param SourceStubber[] $sourceStubbers
+     */
+    public function __construct(array $sourceStubbers = [])
```

Comment:
Can you use `SourceStubber $firstSourceStubber, SourceStubber ...$otherSourceStubbers` here, to avoid an empty list?